.github/conda_envs/minimum-reqs.yml

```
@@ -0,0 +1,32 @@
+name: isce3
+channels:
+  - conda-forge
+  - nodefaults
+dependencies:
+  - backoff
+  - cmake=3.19
+  - gxx_linux-64=11
+  - eigen=3.3
+  - fftw=3.3
+  # GDAL 3.0 is too old to satisfy python>=3.9
+  # GDAL 3.1 is too old to satisfy hdf5>=1.12
+  # GDAL 3.6.0 was the first to build with cmake, needed for FindGDAL module
+  # https://github.com/conda-forge/gdal-feedstock/pull/651
+  - gdal>=3.6
```

Typo? Shouldn't we ensure that this job builds with GDAL 3.6.X (and not some newer GDAL version)?

```suggestion
  - gdal=3.6
```

---

I was utterly unable to get an environment to resolve with gdal 3.6 and had to drop the hard constraint. It may be possible to pin it to a later version that's still less than our latest-build version, not sure.

---

How about we drop the min-reqs job for now, iterate on it some more, and revive it in a subsequent PR?

It looks like the GPU runner and the rest of the CI build matrix are working well -- the min-reqs job is the only holdup from this being merged and proceeding forward with public GitHub migration. The existing internal CI system doesn't test minimum package versions, so there's no regression if we remove it for now.

Some more iteration could let us re-structure the min-reqs job as multiple jobs, each targeting a different dependency (or set of dependencies).

WDYT?

---

Yeah I think that's a good idea. Maybe the min-reqs could be structured as you proposed above - one min-reqs jobs for each dependency we want to enforce a minimum version on. In any case, I agree that we can cook on this a little longer, while the rest of the CI matrix is already in a good place.